python/sglang/srt/cpu_utils.py

```diff
@@ -2,6 +2,8 @@
 
 from typing import TYPE_CHECKING
 
+import torch
+
 if TYPE_CHECKING:
     from sglang.srt.configs.model_config import ModelConfig
 
```

Comment:
add a global attribute `is_intel_amx_backend_available` and try
`from sgl_kernel.ops._kernels import convert_weight_packed`

if succeeded which means that cpu csrc files are compiled successfully, is_intel_amx_backend_available = true. Vise vesa.

Reply:
Fixed in https://github.com/mingfeima/sglang/pull/8/commits/533027853f5681629db97387d1d693d653808bbf